azure-mongodb.yml

```diff
@@ -46,6 +46,14 @@ plans:
     enable_automatic_failover: true
 provision:
   plan_inputs:
+  - field_name: private_endpoint_subnet_id
```

Comment:
Wouldn't it make more sense that the private_endpoint and private_dns fields are part of the plan_inputs instead of user_inputs? These fields are not something that a developer will change or modify since it requires the SPN that the broker is using to have access to these resources. I see these fields being similar to azure subscription, resource SKUs, etc.

It is also under the plan_inputs section in the azure-redis.yml as a reference.

Reply:
That is a good point. Makes sense to leave it as is